<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>668</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smartवडा</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>sw_icon.png</normaloff>sw_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="signinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>641</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1366</width>
      <height>768</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit,QGroupBox,QPushButton
{
      background:transparent;
      color: rgb(23, 109, 2);
      border: 2px solid gray;
      border-radius: 10px;
	  padding: 1px left;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="sw_logo_">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>311</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
border-radius:0px;
border-image:url(:/pic/sw_logo.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>195</y>
       <width>196</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
color: rgb(223, 101, 21);
    </string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;REGISTER NEW WARD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="wardno">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>270</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <property name="placeholderText">
      <string>Ward No.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="address">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>315</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <property name="placeholderText">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="state">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>315</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>State</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phoneNo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Phone No</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="email">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>405</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <property name="placeholderText">
      <string>E-mail ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mun_logo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Municipality/VDC Logo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>495</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="confirmpassword">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>495</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Re-Type your Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browse">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>450</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
border-radius:0px;
border-image:url(:/pic/browse.png);

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="signup">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>555</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #8f8f91;
border-radius: 6px;
background:blue;
color:white;
</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="municipality">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>(Rural) Municipality/(Sub) Metropolitan City</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>municipality</tabstop>
  <tabstop>wardno</tabstop>
  <tabstop>address</tabstop>
  <tabstop>state</tabstop>
  <tabstop>phoneNo</tabstop>
  <tabstop>email</tabstop>
  <tabstop>mun_logo</tabstop>
  <tabstop>browse</tabstop>
  <tabstop>password</tabstop>
  <tabstop>confirmpassword</tabstop>
  <tabstop>signup</tabstop>
 </tabstops>
 <resources>
  <include location="sw.qrc"/>
 </resources>
 <connections/>
</ui>
